lib/jsdom/living/aborting/AbortController-impl.js

```diff
@@ -1,12 +1,19 @@
 "use strict";
 
+const idlUtils = require("../generated/utils");
+const AbortSignal = require("../generated/AbortSignal");
+
 class AbortControllerImpl {
-  constructor(args, privateData) {
-    this.signal = privateData.AbortSignal.createImpl([]);
+  constructor(globalObject) {
+    this._signal = AbortSignal.createImpl(globalObject, []);
+  }
+
+  get signal() {
+    return idlUtils.wrapperForImpl(this._signal);
```

Comment:
This shouldn't be necessary, in theory; the generated wrapper should unwrap the signal.